dist/src/main/java/io/camunda/application/commons/rest/MappingJackson2HttpMessageConverterConfiguration.java

```diff
@@ -27,18 +28,20 @@ public class MappingJackson2HttpMessageConverterConfiguration {
 
   @Bean
   @Order(1)
+  @ConditionalOnRestGatewayEnabled
   public MappingJackson2HttpMessageConverter gatewayRestMappingJackson2HttpMessageConverter(
-      @Lazy @Qualifier("gatewayRestObjectMapper") final ObjectMapper objectMapper) {
+      @Qualifier("gatewayRestObjectMapper") final ObjectMapper objectMapper) {
     final PackageSpecificJackson2HttpMessageConverter messageConverter =
-        new PackageSpecificJackson2HttpMessageConverter("io.camunda.zeebe");
+        new PackageSpecificJackson2HttpMessageConverter("io.camunda.zeebe.gateway.protocol.rest");
```

Comment:
I noticed this mapper was also picking up types for the actuator endpoints from the `io.camunda.zeebe.shared` package, thus providing a more rest API specific package.